frontend/src/pages/TracesExplorer/Filter/SectionContent.tsx

```diff
@@ -92,17 +92,21 @@ export function SectionBody(props: SectionBodyProps): JSX.Element {
 		if (checked) {
 			addFilter(type, newValue, setSelectedFilters, keys);
 			setCheckedItems((prev) => {
-				if (!prev.includes(newValue)) {
-					prev.push(newValue);
+				const arr = prev || [];
+				if (!arr.includes(newValue)) {
```

Comment:
string can still go inside the if block ?

Reply:
done, added isArray check